xbps-mini-builder

```diff
@@ -50,13 +50,13 @@ fi
 
 # Main builder section
 packages="$(cat ../packages.list)"
-xbulk ${packages}
 
-# Sign built packages
-xbps-rindex --sign --signedby "XBPS-mini-builder" --privkey ../id_rsa hostdir/binpkgs
-xbps-rindex --sign-pkg --privkey ../id_rsa hostdir/binpkgs/*.xbps
+xbulk ${packages} "$@"
 
-if [ -d hostdir/binpkgs/nonfree ] ; then
-    xbps-rindex --sign --signedby "XBPS-mini-builder" --privkey ../id_rsa hostdir/binpkgs/nonfree
-    xbps-rindex --sign-pkg --privkey ../id_rsa hostdir/binpkgs/nonfree/*.xbps
-fi
+# Sign built packages
+for folder in {binpkgs binpkgs/multilib binpkgs/multilib/nonfree binpkgs/nonfree}; do
```

Comment:
move binpkgs to be part of the common path and then iterate on the remaining sub-directories.